### DISCUSSION TITLE
ReplyButton action

### DISCUSSION THREAD

**Discussion by uzdeveloper1** [def-studio/telegraph#109] [Q&A]
For ReplyButton not working action

**Comment by fabio-ivona** [comment]
could you show me your code?

**Comment by uzdeveloper1** [comment]
<img width="1792" alt="Screen Shot 2022-06-22 at 17 04 29" src="https://user-images.githubusercontent.com/65232715/175024817-8d5d5f1d-9495-4eb6-9875-a14fc9633514.png">

**Comment by uzdeveloper1** [comment]
<img width="1344" alt="Screen Shot 2022-06-22 at 17 10 59" src="https://user-images.githubusercontent.com/65232715/175025836-1171116f-507b-46be-9379-5f474d8ecb90.png">

**Comment by fabio-ivona** [comment]
It is a telegram limitation: actions are supported only for message keyboards (inline) :

https://core.telegram.org/bots/api/#inlinekeyboardbutton

No actions for reply keyboard buttons, though:

https://core.telegram.org/bots/api/#keyboardbutton

That's why I had to split Button and ReplyButton implementations. 

In order to achieve your feature, you should listen for the button text, that is sent when the user clicks on the button
